Kickstarter-iOS/DataSources/MessageThreadsDataSource.swift

```diff
@@ -7,14 +7,16 @@ internal final class MessageThreadsDataSource: ValueCellDataSource {
     case messageThreads
   }
 
+  internal let emptyStateCellIdentifier = String(describing: MessageThreadEmptyStateCell.self)
```

Comment:
hey @stephencelis in lieu of a more standardized way of accessing these loose cell names across the codebase, I opted for this constant. Let me know if this works.